openedx_external_enrollments/tests/external_enrollments/tests_salesforce_external_enrollment.py

```diff
@@ -230,12 +239,17 @@ def test_get_courses_data(self, get_course_mock, get_date_mock, get_salesforce_m
         course_mock.display_name = 'test-course'
         course_mock.other_course_settings = {'salesforce_data': {}}
         get_course_mock.return_value = course_mock
-        get_date_mock.return_value = now
+        course_key_mock = Mock()
+        course_key_mock.org = 'PX'
+        course_key_mock.course = "test-course-run-id"
+        get_course_key_mock.return_value = course_key_mock
+        get_date_mock.return_value = now.strftime('%Y-%m-%d')
```

Comment:
Instead of typing the date format over and over, you can create a new variable and use this variable instead.  @ivanvgh

Reply:
Done.